src/postData.js

```diff
@@ -0,0 +1,16 @@
+export default async function postTheData() {
+  const scoreForm = document.querySelector('.score-form');
+
+  const formData = new FormData(scoreForm);
+  const data = Object.fromEntries(formData);
+
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/RP7P3DgAooG12Jvz9hXa/scores', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+
+  return response.json();
+}
```

Comment:
- [ ] Ouch! this is really close but once user data is submitted it's not reflected on the scoreboard, the refresh button has to be clicked to see it on the scoreboard. This leads to bad UX as the user might think their score wasn't added properly so please make sure to fix this. ✔

Reply:
Hi, @AbelG101, This was done according to the specifications given [here](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md). That only when the refresh button is clicked that it when the scoreboard shows the updated scores with the new user score.